litmus-portal/frontend/src/pages/EditSchedule/index.tsx

```diff
@@ -0,0 +1,504 @@
+                  </div>
+                  <div className={classes.yamlFlex}>
+                    {weights.length === 0 ? (
+                      <Typography>
+                        {' '}
+                        {t('createWorkflow.verifyCommit.errYaml')}{' '}
+                      </Typography>
+                    ) : (
+                      <Typography>
+                        <b>{yamlStatus}</b>{' '}
+                        {t('createWorkflow.verifyCommit.youCanMoveOn')}
+                      </Typography>
+                    )}
+                    <br />
+                    <ButtonFilled
+                      style={{ width: '20%' }}
+                      onClick={handleEditOpen}
+                    >
+                      {t('createWorkflow.verifyCommit.button.viewYaml')}
+                    </ButtonFilled>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          {/* Cancel and Save Button */}
+          <div className={classes.buttonDiv} aria-label="buttons">
+            <ButtonOutlined
+              onClick={() => {
+                history.push({
+                  pathname: `/workflows/`,
+                  search: `?projectID=${projectID}&projectRole=${userRole}`,
+                });
+              }}
+            >
+              Cancel
+            </ButtonOutlined>
+            <ButtonFilled onClick={() => handleNext()}>
+              Save Changes
+            </ButtonFilled>
+          </div>
+
+          <Modal
+            open={open}
+            onClose={handleEditClose}
+            width="60%"
+            modalActions={
+              <ButtonOutlined
+                onClick={handleEditClose}
+                className={classes.closeBtn}
+              >
+                &#x2715;
+              </ButtonOutlined>
+            }
+          >
+            <YamlEditor content={manifest} filename={workflow.name} readOnly />
+          </Modal>
+
+          {/* Finish Modal */}
+          <div>
+            <Modal
+              data-cy="FinishModal"
+              open={finishModalOpen}
+              onClose={handleFinishModal}
+              width="60%"
+              aria-labelledby="simple-modal-title"
+              aria-describedby="simple-modal-description"
+              modalActions={
+                <div data-cy="GoToWorkflowButton">
+                  <ButtonOutlined onClick={handleFinishModal}>
+                    &#x2715;
+                  </ButtonOutlined>
+                </div>
+              }
+            >
+              <div className={classes.modal}>
+                <img src="/icons/finish.svg" alt="mark" />
+                <div className={classes.heading}>
+                  The Schedule
+                  <br />
+                  <span className={classes.successful}>{workflow.name}</span>,
+                  <br />
+                  <strong>is successfully updated</strong>
+                </div>
```

Comment:
add translations, use margin/padding if possible instead of <br>

Reply:
The breaktags are required to add newline to the final modal

If we use only style we'll get
```
A new workflow, <hash> , is successfully created
```

With br we get

```
 A new workflow,
      <hash>
is successfully created
```

Reply:
And its better to use just br instead of wrapping each individual texts into divs and applying styles